contracts/token/ERC721/ERC721Metadata.sol

```diff
@@ -72,6 +84,22 @@ contract ERC721Metadata is Context, ERC165, ERC721, IERC721Metadata {
         _tokenURIs[tokenId] = uri;
     }
 
+    /**
+     * @dev Internal function to get the base token URI.
```

Comment:
We should also explain here that this will be concatenated to form the result of `tokenURI`. I don't want to suggest any specific phrasing because we haven't yet decided on good names for the concepts involved. (Being discussed in another comment in the PR.)